frontend/src/scenes/persons/Person.tsx

```diff
@@ -27,11 +27,11 @@ const { TabPane } = Tabs
 export const scene: SceneExport = {
     component: Person,
     logic: personsLogic,
-    paramsToProps: () => ({ syncWithUrl: true }),
+    paramsToProps: ({ params }) => ({ syncWithUrl: true, id: params._ }), // wildcard is stored in _
 }
 
 export function Person({ id: urlId }: { id?: string } = {}): JSX.Element {
```

Comment:
@mariusandra I could use some help here! I can't figure out how to pass in the `id` parameter (which is the wildcard generated from `/person/*`. I want that parameter populated here, but I can't seem to follow how/where this scene is generated to pass this in.

Reply:
Oh, it's called `_` here. That would fix it, but could you please point me to the point in code where this scene is created with those params?

Reply:
Ahhhhhh! Thanks! 🤦‍♂️